submissions/aderonkeOdebunmi/htmlForm/Registration.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Registration Form</title>
+    </head>
+    <header>
+        <img src="images/everyqlogo3.jpg">
+            <ul>
+     <li><a href="registration.html">Join Us</a></li>
+    <li><a href="../introductionToHtml/productPage.html">Product Page</a></li>
```

Comment:
This here is what page navigation to all pages it. All you need to do is link all the pages you've created in the header, just like you have done here.